src/SQLQueryStress/CommandLineOptions.cs

```diff
@@ -1,47 +1,27 @@
 ﻿using CommandLine;
-using CommandLine.Text;
-using System.Collections.Generic;
 
 namespace SQLQueryStress
 {
     public class CommandLineOptions
     {
-        private readonly HeadingInfo _headingInfo = new HeadingInfo(System.Reflection.Assembly.GetExecutingAssembly().GetName().Name, System.Reflection.Assembly.GetExecutingAssembly().GetName().Version.ToString());
-
-        [Option("c", null,
+        [Option('c', "settingsFile",
                 HelpText = "File name of saved session settings\r\n")]
-        public string SettingsFile = string.Empty;
+        public string SettingsFile { get; set; } = string.Empty;
 
-        [Option("u", null,
+        [Option('u', "unattended",
                 HelpText = "Run unattended (start, run settings file and quit)")]
-        public bool Unattended = false;
+        public bool Unattended { get; set; } = false;
 
-        [Option("r", null, 
+        [Option('r', "results",
                 HelpText = "Autosave results to specified file")]
-        public string ResultsAutoSaveFileName = string.Empty;
+        public string ResultsAutoSaveFileName { get; set; } = string.Empty;
 
-        [Option("d", null,
+        [Option('d', "dbserver",
                 HelpText = "Database Server")]
-        public string DbServer = string.Empty;
+        public string DbServer { get; set; } = string.Empty;
 
-        [Option("t", null,
+        [Option('t', "threads",
                 HelpText = "Number of threads in unattended mode")]
-        public int NumberOfThreads = -1;
-
-        [HelpOption("?", null,
-                HelpText = "Display this help screen")]
-        public string GetUsage()
```

Comment:
What about the nice help screen??

Reply:
HelpOption was removed and replaced with HelpText.AutoBuild. Please see DisplayHelp method on Program.cs